docs/faq.mdx

```diff
@@ -38,6 +38,16 @@ Example of citing IBM Quantum Composer:
 
 > IBM Quantum Composer. 2023. url: https://<span>quantum.ibm.com/composer</span>
 
+## What happened to Python distributions such as Qiskit Aer, Qiskit Nature, and Qiskit Experiments?
+
+Since the release of the Qiskit SDK v1.0, the command `pip install qiskit` will **only** install the [Qiskit SDK](https://github.com/Qiskit/qiskit). This means that some modules that you have used in the past, such as `qiskit-aer` or `qiskit-terra`, now have to be installed separately.
```

Comment:
```suggestion
Since the release of the Qiskit SDK v1.0, the command `pip install qiskit` will **only** install the [Qiskit SDK](https://github.com/Qiskit/qiskit). This means that some modules that you have used in the past, such as `qiskit-aer` or `qiskit-nature`, now have to be installed separately.
```

technically they cant install terra separately bc it doesnt exist anymore